SourceX/storm/storm.cpp

```diff
@@ -33,6 +34,10 @@ static Mix_Chunk *SFileChunk;
 
 void GetBasePath(char *buffer, size_t size)
 {
+#ifdef PLATFORM_CTR
+	char path[] = "file:sdmc:/3ds/devilutionx/\0";

```

Comment:
C-string literals are implicitly null-terminated, no need for `\0` (that actually results in two `\0`s)